<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseWindow</class>
 <widget class="QWidget" name="ChooseWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>333</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>333</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>333</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Choose window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F7F6F1;</string>
  </property>
  <widget class="QLineEdit" name="inputXValue">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>99</y>
     <width>400</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 239, 213);
color: rgb(60, 60, 60);
font-size: 24px; 
border: 0px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="inputFunction">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 239, 213);
color: rgb(60, 60, 60);
font-size: 24px; 
border: 0px;</string>
   </property>
   <property name="text">
    <string>Input function</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="outputResult">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 239, 213);
color: rgb(60, 60, 60);
font-size: 24px; 
border: 0px;</string>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="xEqualText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>301</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 30px; 
border: 0px;</string>
   </property>
   <property name="text">
    <string>X eqals</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="buttonX">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 127, 80);
font-size: 28px;
border: 0px;
</string>
   </property>
   <property name="text">
    <string>Calculate with X</string>
   </property>
  </widget>
  <widget class="QLabel" name="slider">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>149</y>
     <width>700</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(60, 60, 60);
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="buttonPlot">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>153</y>
     <width>301</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 127, 80);
font-size: 30px;
border: 0px;
</string>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QLabel" name="definitionArea">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>153</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
border: 0px;</string>
   </property>
   <property name="text">
    <string>Definition area</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="xFrom">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>178</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
border: 0px;</string>
   </property>
   <property name="text">
    <string>From:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="xTo">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>178</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
border: 0px;</string>
   </property>
   <property name="text">
    <string>To:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="alowableArea">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>203</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
border: 0px;</string>
   </property>
   <property name="text">
    <string>Alowable area:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="yFrom">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>228</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
border: 0px;</string>
   </property>
   <property name="text">
    <string>From:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="yTo">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>228</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(60, 60, 60);
font-size: 24px; 
border: 0px;</string>
   </property>
   <property name="text">
    <string>To:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="xStartArea">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>178</y>
     <width>102</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 239, 213);
color: rgb(60, 60, 60);
font-size: 20px; 
border: 0px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="xEndArea">
   <property name="geometry">
    <rect>
     <x>599</x>
     <y>178</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 239, 213);
color: rgb(60, 60, 60);
font-size: 20px; 
border: 0px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="yStartArea">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>228</y>
     <width>102</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 239, 213);
color: rgb(60, 60, 60);
font-size: 20px; 
border: 0px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="yEndArea">
   <property name="geometry">
    <rect>
     <x>599</x>
     <y>228</y>
     <width>102</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 239, 213);
color: rgb(60, 60, 60);
font-size: 20px; 
border: 0px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCalculate">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>253</y>
     <width>700</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font-size: 30px;
	background-color: rgb(255, 127, 80);
	border: 0px;
	border-radius: 12px;
}

QPushButton:pressed {
	color: rgb(255, 127, 80);
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
